docs/UserGuide.md

```diff
@@ -988,18 +1108,19 @@ You may want to **re-enter your client data in a fresh JSON file** in the event
+| `p/Phone`                                                             | <ul><li>Should only contain numbers, and should be at least 3 digits long.</li><li>While we disallow usage of symbols such as +, if you wish to use country codes, a reasonable work-around is to omit using of symbols. E.g., to input +6590215365 you may simply type in 6590215635</li><li>Spaces are also not allowed. However, a simple work-around for this is to omit using such spaces. E.g. to input 9021 5365 we can simply type in 90215365.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                     | <ul><li>✅ <code>i/1234</code></li><li>❌ <code>i/123A</code></li></ul>                                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                                                                  |
+| `i/Income`                                                            | <ul><li>**Integer** and should be **at least 0**. </li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | <ul><li>✅ <code>i/10000</code></li><li>❌ <code>i/10,000</code></li><li>❌ <code>i/-1</code></li></ul>                  |
```

Comment:
Actually should we add a constraint for income smth like for phone, if they want to put commas or spaces, a work around is to avoid the use of symbols, like they may think `1,000,000` is a valid integer but we wont allow that hence FF bla bla, so maybe can hedge first

Reply:
actl this is there already tho

Reply:
No to justify why we dont allow, give them a workaround